docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -143,6 +143,18 @@ values:
 
 NOTE: if you use `redirect` or `disabled` and have not added a SSL certificate or keystore, your server will not start!
 
+== Additional HTTP Headers
+
+To enable HTTP headers to be sent on every response, add the following properties:
+
+[source, properties]
+----
+quarkus.http.headers.foo=bar
```

Comment:
Hi George - may be for the docs it can make sense to show a known HTTP response header like disabling a cache control for example ?

Reply:
Ah good idea!

Reply:
@sberyozkin I used `X-Content-Type-Options: sniff` as an example, hope that looks better now 😉